docs/styling/how-to-add-custom-styles.md

```diff
@@ -0,0 +1,89 @@
+# How to override styles
+
+To customize or override a Page Builder style assigned to a content type, we recommend creating a CSS selector that targets the content type by using a custom class, as follows:
+
+-  `html-body` id
+-  `data-content-type` attribute
+-  your custom CSS class
+
+For example, if you wanted to override the Heading content type, the selector for your CSS class should follow this pattern:
+
+![Page Builder style selector](../images/pagebuilder-style-override-selector.svg)
+
+This CSS selector results in a CSS specificity value of 120, which overrides the 110 specificity for Page Builder styles, while keeping the specificity relatively low for any additional overrides, as needed.
```

Comment:
I really like that you are adding specificity value to make it more understandable!